tools/i18n/unescape-html-in-json-translations

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+set -ex
+
+find ./locale \
+    -regextype sed \
+    -regex '^\./locale/.*/\(mobile\|translations\).json$' \
+    -exec ./tools/i18n/unescape-contents --html {} \;
```

Comment:
Did you write a wrapper script to get `set -x` output, or to avoid doing this bit of work in Python?

I think you could do this loop pretty well, either using `os.walk` or using the strategies that `makemessages` / `compilemessages` use...

Reply:
I wrote the wrapper script because the sequencing over time here was:

- Doodle around in the shell while drafting the underlying `unescape-contents`, using `find(1)` mostly because I know it well and had been designing `unescape-contents` around a 1:1 mapping of input file to output file (to make it easier to test a single file at a time, mostly)
- A version of this find block was in `sync-translations`, then got ripped out to this standalone script and modified eventually into what you see here
- This block was then copied back into `sync-translations` yesterday after discussion

I hold a weak opinion here that spinning up a new process per file keeps the implementation a bit tidier (if slightly wall-clock slower) Python-side, and that find's implicit "keep going if a child process for one input fails" behavior is a nice touch, but the latter case can be handled cleanly in Python-land too with a quick few lines.

On the other hand I see the mobile translation scripts do a very similar regex search Python-side, and stop execution of the entire pipeline immediately when finding an error.

Reply:
I think given that most errors require human intervention, the existing model is probably better, but I don't think it matters much. The main downside of using this code for the loop is just that it's different, and thus might behave differently in unusual circumstances.